cmd/operator-sdk/internal/genutil/openapi.go

```diff
@@ -54,41 +121,31 @@ func OpenAPIGen() error {
 	fqApis := k8sutil.CreateFQAPIs(apisPkg, gvMap)
 	f := func(a string) error { return openAPIGen(a, fqApis) }
 	if err = generateWithHeaderFile(f); err != nil {
-		return err
+		return nil, nil, err
 	}
 
-	s := &scaffold.Scaffold{}
 	cfg := &input.Config{
 		Repo:           repoPkg,
 		AbsProjectPath: absProjectPath,
 		ProjectName:    filepath.Base(absProjectPath),
 	}
 	crds, err := k8sutil.GetCRDs(scaffold.CRDsDir)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return cfg, crds, nil
+}
+
+func doScffolding(g, v, k string, s *scaffold.Scaffold, cfg *input.Config) error {
+	r, err := scaffold.NewResource(g+"/"+v, k)
 	if err != nil {
 		return err
 	}
-	for _, crd := range crds {
-		g, v, k := crd.Spec.Group, crd.Spec.Version, crd.Spec.Names.Kind
-		if v == "" {
```

Comment:
Could not just be == before and then, if has the ignore flag;
-> check if the group was informed if not return an error